docs/reference/schema.md

```diff
@@ -10,58 +10,70 @@ This page presents the schema in tables with additional information in paragraph
+- [Loss](#loss-metadata): metadata describing monetary and non-monetary damage and losses produced in a risk assessment. This includes fields to link losses to the hazard, exposure, and vulnerability components used in the analysis. The loss component enables description of common impact and risk metrics for direct and indirect impacts, for individual historical and hypothetical events, and for large events sets in year loss tables and event loss tables.
 
 For general definitions of hazard, exposure, vulnerability and loss, please see the [Glossary](../glossary.md).
 
 For fields that reference [sub-schemas](#sub-schemas), a link is provided to a table with details of the sub-schema. To see how the fields and sub-schemas fit together, consult the [schema browser](browser.md).
 
-The diagram below shows the core relationships between schema components, and their core attributes.
+## Dataset
 
-```{eval-rst}
- .. mermaid::
-
-  classDiagram
-      Dataset -- Hazard
-      Dataset -- Exposure
-      Dataset -- Vulnerability
-      Dataset -- Loss
-      Dataset: -Project name
-      Dataset: -Coverage
-      Dataset: -Purpose
-      Dataset: -Bibliography
-      class Hazard{
-        -Type, Process
-        -Trigger
-        -Frequency, Intensity
-        -Analytical method
-          }
-      class Exposure{
-        -Asset category
-        -Taxonomy
-        -Cost type
-      }
-      class Vulnerability{
-        -Hazard process
-        -Exposure taxonomy
-        -Analytical method
-      }
-      class Loss{
-        -Hazard process
-        -Exposure taxonomy
-        -Loss frequency
-        -Loss metric
-      }          
+The top-level object in the RDLS schema is a risk dataset. A risk dataset is described as:
+
+```{jsoninclude-quote} ../../docs/_readthedocs/html/rdls_schema.json
+---
+jsonpointer: /description
+---
 ```
 
-## Dataset
+The general attributes of a dataset are described by fields based on the [Data Catalog Vocabulary](https://www.w3.org/TR/vocab-dcat-3/) and the [Dublin Core Metadata Initiative Metadata Terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms).
 
-In addition to schema-specific attributes, each dataset is identified by a list of attributes based on the [Dublin Core Metadata Initiative Metadata Terms](https://www.dublincore.org/specifications/dublin-core/dcmi-terms).
+The following diagram shows a subset of dataset-level fields, with required fields highlighted in blue:
+
+```{eval-rst}
+.. uml::
+
+  @startjson
+  <style>
+    .required {
+      BackGroundColor #239ce8
+    }
+  </style>
+  #highlight "title" <<required>>
+  #highlight "risk_data_type" <<required>>
+  #highlight "publisher" <<required>>
+  #highlight "publisher" / "name" <<required>>
+  #highlight "spatial" <<required>>
+  #highlight "license <<required>>
```

Comment:
cross-check fields listed as required in diagram and table - e.g. license is required in table, but not highlighted in diagram

Reply:
Ah, good spot. It was in the list of highlighted fields, but was missing a closing `"`. Fixed in https://github.com/GFDRR/rdl-standard/pull/212/commits/cbd6162f3a2b0835b4bf21e855c2ef761afb5a8a.